codegen/arrai/service.arrai

```diff
@@ -10,6 +10,7 @@ let restService = restClient | {"requestrouter", "servicehandler", "serviceinter
 
 let grpcClient = {"grpc_client"};
 let grpcService = grpcClient | {"grpc_handler", "grpc_interface"};
+let migrate = {"types"};
```

Comment:
I don't think this is necessary. You can just pass through `types.go`.

Reply:
This is part of bigger change where we move more files, anyways they have to move to arrai sooner or later.

Reply:
We'll migrate all generated code in one go, not one generated file at a time. This is a temporary measure to get some confidence that the generated json code is correct.

Reply:
There is another PR open which migrates everything else, with migrate as target. We can leave this here, it will change anyway once you merge that PR.